Z-Wave: set radio frequency region (#39499)
home-assistant/home-assistant.github.io: source/_integrations/zwave_js.markdown

   - Confirm that you connected the new controller by selecting **Submit**.
6. Follow the steps on screen.

## Setting the radio frequency region of the controller in the Z-Wave JS add-on

The frequency used by Z-Wave devices depends on your region. For each country or region, the Z-Wave product has to apply the standard Z-Wave frequency approved for that region. This means that when you buy a Z-Wave product, you need to make sure it is approved for your region.

> If you are using the Z-Wave JS add-on, the region of your Home Assistant is used to determine the radio frequency region. If needed, you can change the radio frequency region for your controller. However, the region of end devices cannot be changed. They only work in the frequency region they were specified for.

Reviewer: ```suggestion
If you are using the Z-Wave JS add-on, the region of your Home Assistant is used to determine the radio frequency region. If needed, you can change the radio frequency region for your controller. However, the region of end devices cannot be changed. They only work in the frequency region they were specified for.
```
```suggestion
If you are using the Z-Wave JS add-on, the country of your Home Assistant is used to determine the radio frequency region. If needed, you can change the radio frequency region for your controller. However, the region of end devices cannot be changed. They only work in the frequency region they were specified for.
```

   - Confirm that you connected the new controller by selecting **Submit**.
6. Follow the steps on screen.

## Setting the radio frequency region of the controller

The frequency used by Z-Wave devices depends on your region. For each country or region, the Z-Wave product has to apply the standard Z-Wave frequency approved for that region. This means that when you buy a Z-Wave product, you need to make sure it is approved for your region. For example, when you live in Europe, it is not allowed to use a device that is using a US frequency.

> During onboarding of your controller, the region of your Home Assistant is used to determine which radio frequency region this controller is using. If needed, you can change the radio frequency region for your controller. However, the region of the device cannot be changed. It only works in the frequency region it was specified for.

Reviewer: This is only applicable for users that are using the Z-Wave JS add-on.

The add-on uses the configured country in Home Assistant to pick the appropriate RF region.

The two last sentences are confusing. What device are we referring to?

> During onboarding of your controller, the region of your Home Assistant is used to determine which radio frequency region this controller is using. If needed, you can change the radio frequency region for your controller. However, the region of the device cannot be changed. It only works in the frequency region it was specified for.

Reviewer 2: ```suggestion
During onboarding of your controller, the region of your Home Assistant is used to determine which radio frequency region this controller is using. If needed, you can change the radio frequency region for your controller. However, the region of end devices cannot be changed. They only work in the frequency region they were specified for.
```

   - Confirm that you connected the new controller by selecting **Submit**.
6. Follow the steps on screen.

## Setting the radio frequency region of the controller in the Z-Wave JS add-on

> The frequency used by Z-Wave devices depends on your region. For each country or region, the Z-Wave product has to apply the standard Z-Wave frequency approved for that region. This means that when you buy a Z-Wave product, you need to make sure it is approved for your region.
> 
> If you are using the Z-Wave JS add-on, the country of your Home Assistant is used to determine the radio frequency region. If needed, you can change the radio frequency region for your controller. However, the region of end devices cannot be changed. They only work in the frequency region they were specified for.

Reviewer 2: Something about these two paragraph bugs me a bit. How about something in this direction?

```suggestion
The frequency used by Z-Wave devices depends on your region. For 700 and 800 series controllers, this frequency can be changed. The frequency of end devices cannot, so you need to make sure to buy devices specific to your region.

If you are using the Z-Wave JS add-on, Home Assistant automatically changes the radio frequency region to match the region/country you're in. If needed, you can override this setting.
```

### Prerequisites

- Administrator rights in Home Assistant
- All your Z-Wave devices must be specified for that region
- Note: this procedure only applies if your controller is [set up using the Z-Wave JS add-on](#setting-up-a-z-wave-js-server)

> ### To change the radio frequency region of your Z-Wave controller

Reviewer 2: Given that we call it override at the beginning of the section, maybe call it the same here?
```suggestion
### Overriding the radio frequency region of your Z-Wave controller
```

1. Go to {% my supervisor_addon addon="core_zwave_js" title="**Settings** > **Add-ons** > **Z-Wave JS**" %}.
2. Open the **Configuration** tab.

> 3. In the **Options** section, select the **Radio Frequency Region**.

Reviewer 2: AFAIK we have an "Automatic" option, which is the default. Should we mention this here?

   - Confirm that you connected the new controller by selecting **Submit**.
6. Follow the steps on screen.

> 

Author: ```suggestion
```
